test/integration/util/util.go

```diff
@@ -98,6 +98,11 @@ func StartScheduler(ctx context.Context, clientSet clientset.Interface, kubeConf
 
 	informerFactory.Start(ctx.Done())
 	informerFactory.WaitForCacheSync(ctx.Done())
```

Comment:
get a logger from context here

Reply:
I added a logger above. But it seems we can't use `ErrorS` with logger

Reply:
yeah, just Error